build.ps1

```diff
@@ -2,3 +2,12 @@
 
 # Set goreleaser to build for current platform only
 goreleaser build --snapshot --rm-dist --single-target
+
+$currentDir = Get-Location
+$dirs = Get-ChildItem dist/ –recurse -directory
+
+git clone -b main --single-branch https://github.com/puppetlabs/baker-round dist/templates
+
+$dirs | ForEach-Object {
+  New-Item -Path "$($_.FullName)/templates" -ItemType SymbolicLink -Value "$($currentDir.Path)/dist/templates"
+}
```

Comment:
```suggestion
[CmdletBinding()]
param (
  [Parameter()]
  [ValidateSet('build', 'package')]
  [string]
  $Target = 'build'
)

switch ($Target) {
  'build' {
    $arch = go env GOHOSTARCH
    $platform = go env GOHOSTOS
    $binPath = Join-Path $PSScriptRoot "dist" "pct_${platform}_${arch}"
    # Set goreleaser to build for current platform only
    goreleaser build --snapshot --rm-dist --single-target
    git clone -b main --single-branch https://github.com/puppetlabs/baker-round (Join-Path $binPath "templates")
  }
  'package' {
    git clone -b main --single-branch https://github.com/puppetlabs/baker-round "templates"
    goreleaser --skip-publish --snapshot --rm-dist
  }
}
```

Reply:
I'm not sure how to replicate this with the bash script, but this prevents having to do symlinks. Creating a symbolic link requires an administrative prompt on Windows, which is a wall to put up in front of a Windows developer. This also handles having to iterate over the build directory to find the right folder to make the symlink to.

Honestly, with all this we have to do here, and the fact that it is temporary, and that this is all handled inside the goreleaser stuff I'm inclined to draw a line in the sand and say the local build script doesn't handle packaging.

Reply:
Thanks for the PS script. I've emulated it in `build.sh`.